business/istio_status.go

```diff
@@ -223,3 +209,39 @@ func GetDeploymentStatus(d apps_v1.Deployment) string {
 	}
 	return status
 }
+
+func (iss *IstioStatusService) getAddonComponentStatus() IstioComponentStatus {
+	extServices := config.Get().ExternalServices
+	ics := IstioComponentStatus{}
+
+	ics.merge(getAddonStatus("prometheus", true, extServices.Prometheus.URL, true))
```

Comment:
It doesn't have to be for that PR, but I'm just thinking we could check also the second prometheus URL (for custom dashboards) in case it is in use.
I can also do it after your PR if you prefer

Reply:
Cool! I can add it, sure.